securesystemslib/signer/__init__.py

```diff
@@ -15,14 +16,14 @@
     SIGNER_FOR_URI_SCHEME,
     SecretsHandler,
     Signer,
-    SSlibSigner,
 )
 from securesystemslib.signer._sigstore_signer import SigstoreKey, SigstoreSigner
 from securesystemslib.signer._spx_signer import (
     SpxKey,
     SpxSigner,
     generate_spx_key_pair,
 )
+from securesystemslib.signer._sslib_signer import SSlibSigner
 
 # Register supported private key uri schemes and the Signers implementing them
 SIGNER_FOR_URI_SCHEME.update(
```

Comment:
Should we change the default handler for FILE_URI_SCHEME? (and maybe leave a comment that ENVVAR_URI_SCHEME is deprecated unless someone reimplements it)